### ISSUE TITLE
Angular Material padding overlay shifts contents up

### ISSUE THREAD

**Issue by nicky-lenaers** [brianpkelley/md-color-picker#45]
For `angular-material@1..0.5`, the `md-tabs` has a `padding-bottom` of `48px`, which shifts the contents up, such that they overlay the RGB/HSL/HEX value.

Suggested CSS correction (just to give you a direction):

```
[md-color-picker-dialog] md-tabs {
    padding-bottom: 0 !important;
}
```

Great library, tanks!

**Comment by brianpkelley** [comment]
Seems the fix has been applied and mentioned in #25

**Comment by nicky-lenaers** [comment]
Great, thanks!
